docs/skopeo-copy.1.md

```diff
@@ -149,6 +149,18 @@ Let's say out of 3 layers that the image `docker.io/library/nginx:1.17.8` is mad
 skopeo  copy --encryption-key jwe:./public.key --encrypt-layer 1 oci:local_nginx:1.17.8 oci:try-encrypt:encrypted
 ```
 
+## NOTES
+
+In the context of `signatures`, this refers to `container signatures` as implemented by the [github.com/containers/image/signature](https://github.com/containers/image/tree/main/signature) package.
+
+```text
```

Comment:
I’m not sure we need this quote — just a reference to the `containers-signature(5)` man page might be enough. In particular “the JSON document” seems to me to be mostly irrelevant to users of `skopeo copy`.

Reply:
Hey @mtrmac yeah, I'd not fully appreciated that these docs are used to generate the `man` pages for `skopeo` and have thus slimmed things down considerably - about to commit ....